aten/src/ATen/native/BatchLinearAlgebra.cpp

```diff
@@ -196,6 +206,14 @@ template<> void lapackSvd<float>(char jobz, int m, int n, float *a, int lda,
                                  float *s, float *u, int ldu, float *vt, int ldvt, float *work, int lwork, int *iwork, int *info) {
   sgesdd_(&jobz, &m, &n, a, &lda, s, u, &ldu, vt, &ldvt, work, &lwork, iwork, info);
 }
+
+template<> void lapackLuSolve<double>(int n, int nrhs, double *a, int lda, int *ipiv, double *b, int ldb, int *info) {
+  dgetrs_("N", &n, &nrhs, a, &lda, ipiv, b, &ldb, info);
```

Comment:
Is this correct? Shouldn't it be `'N'` (the character version?) I'm not sure why this works in string form

Reply:
Oh yes, I’m sorry. I’ll fix this right away.

Reply:
No worries, I'm just really surprised that our builds allow this conversion from string literal to char